api/internal/handler/ssl/ssl_test.go

```diff
@@ -19,146 +19,859 @@ package ssl
 
 import (
 	"encoding/json"
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"net/http"
 	"testing"
-	"time"
 
 	"github.com/shiningrush/droplet"
+	"github.com/shiningrush/droplet/data"
 	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/mock"
 
-	"github.com/apisix/manager-api/internal/conf"
 	"github.com/apisix/manager-api/internal/core/entity"
-	"github.com/apisix/manager-api/internal/core/storage"
 	"github.com/apisix/manager-api/internal/core/store"
+	"github.com/apisix/manager-api/internal/handler"
+	"github.com/apisix/manager-api/internal/utils/consts"
 )
 
-func TestSSL(t *testing.T) {
-	// init
-	err := storage.InitETCDClient(conf.ETCDConfig)
+func getTestKeyCert(t *testing.T) (string, string) {
+	testCert, err := ioutil.ReadFile("../../../test/certs/test2.crt")
 	assert.Nil(t, err)
-	err = store.InitStores()
+	testKey, err := ioutil.ReadFile("../../../test/certs/test2.key")
```

Comment:
If the key pair is not used in other pleases, what about just put them in the `testdata` directory.

Reply:
These two files are also used by E2E test. In order not to create data repeatedly, I directly use the key and cert files.

Reply:
OK